tests/page/elementhandle-misc.spec.ts

```diff
@@ -47,6 +47,14 @@ it('should fill input when Node is removed', async ({ page, server }) => {
   expect(await page.evaluate(() => window['result'])).toBe('some value');
 });
 
+it('should clear input', async ({ page, server }) => {
+  await page.goto(server.PREFIX + '/input/textarea.html');
+  const handle = await page.$('input');
+  await handle.fill('some value');
+  await handle.clear();
+  expect(await page.evaluate(() => window['result'])).toBe('');
```

Comment:
I'd also check that value was not empty before calling `clear()`.

Reply:
Done.